config/network/node.ini

```diff
@@ -4,6 +4,10 @@
 ### Protocols can be 'pnp' (default), 'pnps', 'pamr' (for networks behind firewall)
 proactive.communication.protocol=pnp
 
+proactive.runtime.ping=false
```

Comment:
Could you put comments for this property?

Reply:
ok